lib/index.js

```diff
@@ -31,16 +31,15 @@ internals.schema = Joi.object().keys({
 
 internals.implementation = (server, options) => {
 
-    Hoek.assert(options, 'Missing bearer auth strategy options');
-
+    Hoek.assert(options, new Error('Missing bearer auth strategy options'));
```

Comment:
Yep, this can be done: https://github.com/hapijs/hoek/blob/master/API.md#assertcondition-message. TIL 👍 

Curious whether this or other gives a nicer stack trace, but happy to leave it as is, it's a little clearer in the code.

Reply:
Afaik it removes the assert function call from the stack https://github.com/hapijs/hoek/blob/master/lib/index.js#L737

Reply:
Yes please